semver/_is_comparator_test.ts

```diff
@@ -1,29 +1,27 @@
 // Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
 import { assert } from "../assert/mod.ts";
-import { INVALID, MIN } from "./constants.ts";
+import { MIN } from "./constants.ts";
 import { isComparator } from "./_is_comparator.ts";
+import { formatComparator } from "./_format_comparator.ts";
+import { Comparator } from "./types.ts";
 
 Deno.test({
-  name: "valid_comparator",
+  name: "isComparator()",
   fn: async (t) => {
-    let i = 0;
-    const comparators: unknown[] = [
+    const comparators: Comparator[] = [
       {
         operator: ">=",
-        semver: { major: 0, minor: 0, patch: 0, prerelease: [], build: [] },
-        min: { major: 0, minor: 0, patch: 0, prerelease: [], build: [] },
-        max: { major: 0, minor: 0, patch: 0, prerelease: [], build: [] },
-      },
-      {
-        operator: "<",
-        semver: MIN,
-        min: INVALID,
-        max: INVALID,
+        major: 0,
```

Comment:
Let's try to keep this PR focused on test names. We can rework the tests themselves in a follow-up PR. Ditto to other parts of the PR.

Reply:
reverted